kuma/wiki/models.py

```diff
@@ -1479,18 +1471,9 @@ def zone_stack(self):
 
     def get_share_url(self):
```

Comment:
Do we even need that method in that case? Could we instead call it `get_full_url` so we can use it for other places?